src/Main.hs

```diff
@@ -141,6 +131,9 @@ intersects (Ray origin direction) object@(Object (Sphere center radius) material
             [] -> Nothing
             otherwise -> Just (minimum listOfRoots, object)
 
+deg2Rad :: Double -> Double
```

Comment:
I'll put it in a Util on another commit, I'll add an issue for it. Pretty sure we think in terms of degrees w/r/t FOV, and this isn't too much trouble